recipes/scikit-rf/meta.yaml

```diff
@@ -0,0 +1,63 @@
+
+{% set name = "scikit-rf" %}
+{% set version = "0.14.3" %}
+{% set compress_type = "zip" %}
+{% set hash_type = "md5" %}
+{% set hash_val = "3dc7e7aab5e0dd1f5ce692f32b053ec7" %}
+{% set build_num = "0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.{{ compress_type }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ compress_type }}
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  number: {{ build_num }}
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - pandas
+    - setuptools
+    - ipython
+    - numpy
+    - scipy
+    - matplotlib
+    - mock
+    - nose
+    - future
+    - xlwt
+    - xlrd
+  run:
+    - python
+    - ipython
+    - numpy
+    - scipy
+    - matplotlib
+    - pandas
+
+test:
+  imports:
+    - skrf
+    - skrf.calibration
+    - skrf.data
+    - skrf.io
+    - skrf.media
+
+
+about:
+  home: http://www.scikit-rf.org
+  license: new BSD
```

Comment:
New BSD is the `BSD 2-Clause` or `BSD 3-clause`? (We rather use the clause syntax to keep it easier to identify btw.)